thinkhazard/static/less/report.less

```diff
@@ -274,18 +274,23 @@ footer {
   }
 
   .level-@{level} {
-    .page-header.overview {
-      border-color: #ccc;
-      &:hover {
-        border-color: fade(@@color, 50%);
-        background-color: @@color-overview;
+    .page-header {
+      &.overview {
+        border-color: #ccc;
+        &:hover {
+          border-color: fade(@@color, 50%);
+          background-color: @@color-overview;
+        }
+      }
+      .level {
+        font-weight: 600;
```

Comment:
I'm not in favor of putting properties which are not specific to a level (no color change) in the mixin.